### PR TITLE
Uno v2 update to rules and config

### PR DESCRIPTION
**Pull request by Squaredwaves**
## Description
updates to make v2 work

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ x] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ x] My change requires a change to the documentation.
- [x ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/uno/uno.h [hunk 1]
```diff
@@ -1,24 +1,8 @@
...
-#pragma once
-
-#include "quantum.h"
-
-#define LAYOUT( \
-  K00  \
-) { \
-  { K00 } \
-}
+#pragma once
```
Comment (drashna) [inline keyboards/uno/uno.h:1]: Please re-add the license header.

File: keyboards/uno/v2/config.h [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#pragma once
```
Comment (drashna) [inline keyboards/uno/v2/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/uno/v2/v2.c [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (drashna) [inline keyboards/uno/v2/v2.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/uno/v2/v2.c (hunk 3 above)
Comment (Squaredwaves) [reply keyboards/uno/v2/v2.c]: Added with my name and year. I am new to qmk and pull requests, so I hope I did it right

File: keyboards/uno/v2/config.h (hunk 2 above)
Comment (Squaredwaves) [reply keyboards/uno/v2/config.h:17]: Added with my name and year. I am new to qmk and pull requests, so I hope I did it right

File: keyboards/uno/uno.h (hunk 1 above)
Comment (Squaredwaves) [reply keyboards/uno/uno.h:17]: added

**Review by drashna — APPROVED** [review]

File: keyboards/uno/v2/readme.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# uno
```
Comment (fauxpark) [inline keyboards/uno/v2/readme.md:1]: Please use Unix line endings for all files.

File: keyboards/uno/keymaps/Squaredwaves/readme.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# The default keymap for uno
+![Keymap](https://i.imgur.com/lCPZZpO.png)
```
Comment (fauxpark) [inline keyboards/uno/keymaps/Squaredwaves/readme.md:2]:
```suggestion

![Keymap](https://i.imgur.com/lCPZZpO.png)
```

File: keyboards/uno/keymaps/Squaredwaves/keymap.c [hunk 6]
```diff
@@ -0,0 +1,70 @@
+/* Copyright 2020 Snipeye
```
Comment (fauxpark) [inline keyboards/uno/keymaps/Squaredwaves/keymap.c:1]: Keymap names should be all lowercase.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/uno/v2/readme.md (hunk 4 above)
Comment (Squaredwaves) [reply keyboards/uno/v2/readme.md:1]: Ok, I think I did it. If not please elaborate

File: keyboards/uno/keymaps/Squaredwaves/keymap.c (hunk 6 above)
Comment (Squaredwaves) [reply keyboards/uno/keymaps/Squaredwaves/keymap.c]: done

**Review by fauxpark — CHANGES_REQUESTED** [review]
There are still files with Windows line endings. We can't merge this until it's fixed.

**Comment by Squaredwaves** [comment]
> There are still files with Windows line endings. We can't merge this until it's fixed.

I've re-uploaded everything after running through sublime text that should add all the unix line endings. If this isn't correct I'm lost as to what to do next

**Comment by fauxpark** [comment]
These files are still using Windows line endings (you can see this in the diff, where some of them have no actual changes but the entire file is "changed"):
```
keyboards/uno/readme.md
keyboards/uno/uno.h
keyboards/uno/v1/readme.md
keyboards/uno/v2/readme.md
keyboards/uno/v2/rules.mk
keyboards/uno/v2/v2.c
```
Do not use the GitHub editor to commit files; use `git` on the command line instead.

**Pull request closed without merging**